### PR TITLE
readme: add real example

### PR DESCRIPTION
**Pull request by TomasTomecek**
Since we don't have a demo yet, let's have a real example in our readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,33 @@ Or if the log file is stored locally:
...
+```
```
Comment (xsuchy) [inline README.md:67]: The lines are really loong. What about wrapping them? https://stackoverflow.com/questions/41238148/how-to-wrap-long-lines-inside-of-markdown-code-in-github-and-gitlab-issu

**Comment by TomasTomecek** [comment]
we can strip those mega long lines, indeed they are confusing

File: README.md (hunk 1 above)
Comment (TomasTomecek) [reply README.md:67]: thanks for the suggestion, will wrap

**Comment by TomasTomecek** [comment]
is this more readable?

the reason I sent this PR is that 2 people last week asked me about a demo - I think showing a real example is a good start

**Review by xsuchy — APPROVED** [review]
Much better for me. 
+1 from me.

**Review by jpodivin — APPROVED** [review]

**Pull request merged**